site_preference_builder/main.js

```diff
@@ -68,9 +70,13 @@ async function main() {
   updatedSitePref = addAPIInfo(updatedSitePref, url, apiKey, merchantId);
   updatedSitePref = addOnShipingOption(updatedSitePref, plans);
 
-  writeFile(OUTPUT_FILE, jsonToXML(updatedSitePref));
+  if (process.env.TOGGLE_REFUND === 'on') {
```

Comment:
could we have a const instead of 'on' ?
like
``` 
const REFUND_IS_DISABLED = 'off'
```
we want it to be activated by default so the condition should look like :
```
if (process.env.TOGGLE_REFUND !== REFUND_IS_DISABLED)
```

Reply:
I'd really like to avoid to require our merchant do configuration : 

I'd want the refund feature to be `on` by default and it could be disabled from an .env variable.

that way, most of the merchant won't have anything about the refund in their .env and thoses would need to disable the refund feature could fill something to do so.

To keep this and keep the code simple we could have :
- in the .env
```
DISABLE_REFUND=true
```
and in the code :
```
if (process.env.DISABLE_REFUND == 'true')
```
@syjust-alma do you agree :D ?